templates/element/sidebar.php

```diff
@@ -9,22 +9,22 @@
 }
 ?>
 
-<div class="collapse navbar-collapse navbar-ex1-collapse navbar-left bs-sidebar">
-    <nav>
-        <ul class="nav nav-pills nav-stacked">
+<div class="collapse navbar-collapse show navbar-ex1-collapse navbar-left bs-sidebar">
+        <ul class="nav nav-pills flex-column bg-light">
```

Comment:
Any particular reason the `<nav>` tag was dropped?

Reply:
It made no difference that I could see, ul.nav seems sufficient. I'll add it back though.